vestings/csv_parser.py

```diff
@@ -2,14 +2,14 @@
 from database import VestingModel
 from constants import *
 import csv
-import datetime
+from dateutil.parser import parse
 from vesting import Vesting
 from web3 import Web3
 
 
 def parse_vestings_csv(db: orm.Session, type, chain_id, verbose):
 
-    vesting_file = f"assets/{chain_id}/{type}_airdrop.csv"
+    vesting_file = f"assets/{chain_id}/{type}_vestings.csv" if type == "investor" else f"assets/{chain_id}/{type}_airdrop.csv"
```

Comment:
There is a lot of "if" with the type. Why not define these as a mapping and then access them via the key. This can even be extended to throw a meaningful error if the type is not found.